projects/plugins/jetpack/modules/stats.php

```diff
@@ -198,6 +198,28 @@ function stats_template_redirect() {
 		}
 	}
 
+	/**
+	 * Allow excluding specific IP addresses from being tracked in Stats.
+	 * Note: for this to work well, visitors' IP addresses must:
+	 * - be stored and returned properly in IP address headers;
+	 * - not be impacted by any caching setup on your site.
+	 *
+	 * @module stats
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param array $excluded_ips An array of IP address strings to exclude from tracking.
+	 */
+	$excluded_ips = apply_filters( 'jetpack_stats_excluded_ips', array() );
```

Comment:
```suggestion
	$excluded_ips = (array) apply_filters( 'jetpack_stats_excluded_ips', array() );
```

We can ensure this is an array and avoid anyone doing_it_wrong running into issues with the `in_array` later.